app/src/main/res/navigation/nav_graph.xml

```diff
@@ -1,35 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
There should be the same indentations as there were before.

Reply:
Done

Reply:
Unfortunately, it is not ;).